pom.xml

```diff
@@ -125,7 +125,7 @@ flexible messaging model and an intuitive client API.</description>
       <dependency>
         <groupId>org.asynchttpclient</groupId>
         <artifactId>async-http-client</artifactId>
-        <version>2.0.31</version>
+        <version>2.1.0-alpha26</version>
```

Comment:
we may want to add following properties in [ahc.properties](https://github.com/apache/incubator-pulsar/blob/master/pulsar-client/src/main/resources/ahc.properties) or just copy `ahc-default.properties` file from `2.1.0-alpha26` to `pulsar-client`.
```
org.apache.pulsar.shade.org.asynchttpclient.aggregateWebSocketFrameFragments=true
org.apache.pulsar.shade.org.asynchttpclient.useLaxCookieEncoder=false
org.apache.pulsar.shade.org.asynchttpclient.useInsecureTrustManager=false
org.apache.pulsar.shade.org.asynchttpclient.acceptAnyCertificate=false
org.apache.pulsar.shade.org.asynchttpclient.disableHttpsEndpointIdentificationAlgorithm=false
org.apache.pulsar.shade.org.asynchttpclient.httpClientCodecInitialBufferSize=128
```

Reply:
Oh, sure, should I then copy the properties file and modify the package name, right?

Reply:
yes, that would be better.